lib/ret_web/router.ex

```diff
@@ -4,12 +4,20 @@ defmodule RetWeb.Router do
   use Sentry.Plug
 
   pipeline :secure_headers do
-    plug(:put_secure_browser_headers)
+    plug(:put_secure_browser_headers, %{
+      "cross-origin-opener-policy" => "same-origin",
+      "cross-origin-resource-policy" => "require-corp"
```

Comment:
Pretty sure this is wrong, and you actually meant to use the embedder-policy header, not the resource-policy header.

```suggestion
      "cross-origin-embedder-policy" => "require-corp"
```